### PR TITLE
Add python-xattr

### PR DESCRIPTION
**Pull request by balrog-nona**
Python component for youtube-dl (https://github.com/OpenIndiana/oi-userland/pull/5569)

### CODE REVIEW

File: components/python/xattr/xattr-PYVER.p5m [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)-$(PYV)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
```
Comment (Mno-hime) [inline components/python/xattr/xattr-PYVER.p5m:24]: xattr seems to [require CFFI](https://github.com/xattr/xattr/blob/master/requirements.txt), probably add it to this file somewhere?

File: components/python/xattr/Makefile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+BUILD_BITS=		64
+BUILD_STYLE=		setup.py
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		xattr
+COMPONENT_VERSION=	0.9.7
+COMPONENT_SUMMARY=	'Extended attributes library wrapper for Python'
+COMPONENT_PROJECT_URL=	https://pyxattr.k1024.org/
```
Comment (Mno-hime) [inline components/python/xattr/Makefile]: This page holds outdated information. https://github.com/xattr/xattr seems to be a more viable one.

File: components/python/xattr/xattr-PYVER.p5m [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+file path=usr/bin/xattr
```
Comment (Mno-hime) [inline components/python/xattr/xattr-PYVER.p5m]:
```
 newman  ~  touch xxx
 newman  ~  /usr/bin/xattr xxx 
Traceback (most recent call last):
  File "/usr/bin/xattr", line 11, in <module>
    load_entry_point('xattr==0.9.7', 'console_scripts', 'xattr')()
  File "/usr/lib/python3.5/vendor-packages/xattr/tool.py", line 196, in main
    attr_value = attr_value.decode('utf-8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd3 in position 36: invalid continuation byte
```

Comment (balrog-nona) [reply components/python/xattr/xattr-PYVER.p5m]: I can't fix it and youtube-dl doesn't require it for standard usage.

**Pull request closed without merging**